releasenotes/notes/parameterized_target_c-9c9a4ad2915c2348.yaml

```
@@ -0,0 +1,24 @@
+---
+features_c:
+  - |
+    The ``QkTarget`` type is now able to represent targets that support
+    parameterized ``QkGate`` types that accept any parameter value. Previously,
+    these gates could only be added to the target with a fixed angle value
+    supported by using ``qk_target_entry_new_fixed`` to create the ``QkTargetEntry``.
+    Now, the ``qk_target_entry_new`` function can be used with parameterized gates
+    and when it is this indicates the gate in the target will support any value for
+    all of the gate's parameters. For example:
+
+    .. code-block:: c
+
+        #include <qiskit.h>
+
+        QkTarget *target = qk_target_new(5);
+        QkTargetEntry *rz_entry = qk_target_entry_new(QkGate_RZ);
+        for (uint32_t i = 0; i < 5; i++) {
+            uint32_t qargs[1] = {i};
+            result_prop = qk_target_entry_add_property(rz_entry, qargs, 1, 1.2e-6, 1.3e-9);
+        }
```

This example is incomplete and has an error, these modifications will help.
```suggestion
        QkTarget *target = qk_target_new(5);
        QkTargetEntry *rz_entry = qk_target_entry_new(QkGate_RZ);
        for (uint32_t i = 0; i < 5; i++) {
            uint32_t qargs[1] = {i};
            qk_target_entry_add_property(rz_entry, qargs, 1, 1.2e-6, 1.3e-9);
        }
        qk_target_add_instruction(target, rz_entry);

        // Clean up after using target
        qk_target_free(target);
```